xml/tuning-architecture.xml

```diff
@@ -327,6 +324,13 @@
          100% overhead to allow for maintenance takes the space allocation
          suggested for the first three tiers to 60&nbsp;GB.
        </para>
+       <note>
+         <para>
+           Keep in mind that we recommend reserving 4&nbsp;GB for the WAL device. The recommended
+           size for DB is a total of 64&nbsp;GB for most workloads. See <xref linkend="rec-waldb-size"/>
+           for more information.
+         </para>
+       </note>
        <para>
          Making the decision to provision fast space for the fourth tier of
          RocksDB is something completely related to the expected metadata
```

Comment:
```suggestion
         RocksDB is entirely related to the expected metadata
```

Reply:
Also in the rest of the para...

> somewhere in a mild

somewhere *from* mild to moderate

> whether the fourth tier makes sense to move to faster media.

whether it makes sense to move the fourth tier to faster media.